templates/swagger/v1_json.tmpl

```diff
@@ -5977,14 +6212,14 @@
           },
           {
             "type": "integer",
-            "description": "page wants to load",
+            "description": "page number of results to return (1-based)",
             "name": "page",
             "in": "query"
           },
           {
             "type": "integer",
-            "description": "items count every page wants to load",
-            "name": "per_page",
```

Comment:
Can you have `per_page` be backwards compatible, as well as mark it as deprecated in swagger?